pkgs/misc/emulators/citra/default.nix

```diff
@@ -1,14 +1,16 @@
-{ mkDerivation, lib, fetchgit, cmake, SDL2, qtbase, qtmultimedia, boost }:
+{ mkDerivation, lib, fetchgit, cmake, SDL2, qtbase, qtmultimedia, boost, fetchFromGitHub  }:
 
 mkDerivation {
   pname = "citra";
-  version = "2020-12-07";
+  version = "2021-07-01";
 
   # Submodules
-  src = fetchgit {
-    url = "https://github.com/citra-emu/citra";
-    rev = "3f13e1cc2419fac837952c44d7be9db78b054a2f";
-    sha256 = "1bbg8cwrgncmcavqpj3yp4dbfkip1i491krp6dcpgvsd5yfr7f0v";
+  src = fetchFromGitHub {
+    owner = "citra-emu";
+    repo = "citra";
+    rev = "afed4953bccfb9b39558cf196492c2558ca1f109";
+    sha256 = "11sicbgxa54wwj3jrklprqiznpxjqmxrymz23lvccyybq3pnmfqj";
+    fetchSubmodules = true;
```

Comment:
I don't like that we vendor so many projects.

Reply:
we have zstd, fmt, libusb and probably more in nixpkgs

Reply:
Sorry I don't really understand what you're asking me to do here.

Reply:
Ideally we don't want to fetchSubmodules at all. If that is not possible than we should remove all projects which we have in nixpkgs via rm in postPatch and provide our versions of those libraries.